app/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/onboarding/EmailVerificationService.scala

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.cgtpropertydisposalsfrontend.services
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.services.onboarding
```

Comment:
email verification is not just onboarding - it's also account as well

Reply:
fixed